gnuradio-stable.lwr

```diff
@@ -34,4 +34,7 @@ config:
       gitbranch: UHD-3.15.LTS
     gqrx:
       forcebuild: True
-
+    libvolk:
+      gitbranch: maint
```

Comment:
I just stumbled across your PR. `maint` is not an active branch for VOLK. Please point to a release tag. I'd recommend to point to `v2.4.1` at the moment.  It works well with gr3.8 and gr3.9.

Reply:
@jdemel
Supposedly gnuradio-stable.lwr is for gr3.7 build. Is there a tag that you recommend?

Reply:
With GR 3.7 I'd recommend to not point to VOLK at all because at that point VOLK was still a submodule of GR.
If you'd build GR 3.7 from source, you'd clone it recursively and obtain the correct VOLK source copy. If you'd use a package manager and VOLK is already an independent package, it'll install VOLK automatically.

Besides, the move to version 2 was made with the intention to use SemVer but did not actually introduce breaking changes. This might be different with future major version releases though.
Anyways, I'd recommend v2.4.1 for now for source installs. If we'd receive reports that smth is broken, we could go back to an earlier tag.